src/components/DaccManager/DaccManager.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react'
+
+import { isQueryLoading, useClient, useQuery } from 'cozy-client'
+
+import { SETTINGS_DOCTYPE } from 'src/doctypes'
+import { buildSettingsQuery } from 'src/queries/queries'
+import DaccBanner from 'src/components/DaccBanner/DaccBanner'
+import DaccCompareDialog from 'src/components/DaccManager/DaccCompareDialog'
+import DaccPermissionsDialog from 'src/components/DaccManager/DaccPermissionsDialog'
+import DaccReasonsDialog from 'src/components/DaccManager/DaccReasonsDialog'
+
+const DaccManager = () => {
+  const [showCompareDialog, setShowCompareDialog] = useState(false)
+  const [showPermissionsDialog, setShowPermissionsDialog] = useState(false)
+  const [showReasonsDialog, setShowReasonsDialog] = useState(false)
+  const client = useClient()
+  const settingsQuery = buildSettingsQuery()
+  const { data: settings, ...settingsQueryLeft } = useQuery(
+    settingsQuery.definition,
+    settingsQuery.options
+  )
+
+  const isLoading = isQueryLoading(settingsQueryLeft)
+
+  if (isLoading) {
+    return null
+  }
+
+  const { hideDaccAlerter, allowSendDataToDacc } = settings[0]
```

Comment:
nit: Pas sûr que cela puisse exister, mais ça va casser si `settings` est vide
```suggestion
  const { hideDaccAlerter, allowSendDataToDacc } = settings[0] || {}
```